backend/db.ts

```diff
@@ -97,12 +109,19 @@ function getDbModule(dbConfig: DbConfig, loggerFactory: LoggerFactory, modelFact
         function getHostname(dbUrl: string) {
             return new URL(dbUrl).hostname;
         }
+        function getCertificateAuthority() {
+            const { dialectOptions } = dbConfig.options;
+            return dialectOptions?.ssl ? readFileSync(dialectOptions.ssl.ca, { encoding: 'utf8' }) : undefined;
+        }
         function isResponding(url: string) {
             const patroniUrl = `https://${getHostname(url)}:8008`;
-            return axios(patroniUrl)
+            const ca = getCertificateAuthority();
+            const config = ca ? { httpsAgent: new https.Agent({ ca }) } : undefined;
+
+            return axios(patroniUrl, config)
                 .then(response => response.status === 200)
                 .catch(error => {
-                    logger.debug(`Error occured when requesting: ${url}.`, error);
+                    logger.error(`Error occured when requesting: ${patroniUrl}.`, error);
```

Comment:
I have changed from `debug` to `error`, but I'm not fully sure about changing log level here. It's just a check if DB host is responding. On the other hand - with default logs configuration (in `/etc/cloudify/logging.conf` it's set to WARNING) we don't see anything and any kind of `npm run db-migrate*` command failing at DB host selection is just "hanging" in the CLI.

What do you think?